examples/stable_diffusion_v2/ldm/modules/train/ema.py

```diff
@@ -32,7 +33,10 @@ def __init__(self, network, ema_decay=0.9999, updates=0, trainable_only=True):
 
         self.hyper_map = C.HyperMap()
         self.map = ops.HyperMap()
-        self.assign = ops.Assign().add_prim_attr("primitive_target", "CPU")
+        if offloading:
+            self.assign = ops.Assign().add_prim_attr("primitive_target", "CPU")
```

Comment:
the last time I used offloading was in this way:
self.conv = nn.Conv2d(...)
self.conv.conv2d.add_prim_attr("primitive_target", "CPU")
Not sure if there's a difference.

Reply:
Why `self.conv.conv2d.add_prim_attr`  rather than `self.conv.add_prim_attr`

Reply:
an nn.Cell would suffice